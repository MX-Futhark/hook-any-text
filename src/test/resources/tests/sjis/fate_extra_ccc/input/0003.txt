815c815c815c82c68142004332333631393630303396f693b488ea90ac234344040000001415c90800007041000084420000803f0000803f00000000000020002d0000002800000080000000ffffffff0002000000010000ffffffff0f000000180000006014c908d013c9089c42a34800000000000000008cc482ce82ea82bd8b4382aa82b582c491ab82f091ac82df82e982c681410a82bb82b182c982cd90b6936b89ef92b782c582e082a082e89746906c82c582e082a082e981410a235255425382e882e382a482c782a482a282c182b982a22352554245234332333631393630303396f693b488ea90ac23434445462352454e4482aa97a782c182c482a282bd814200c90800000000000100000002000000030000000400000005000000060000000700000008000000090000000a0000000b0000000c0000000d0000000e0000000f000000100000001100000012000000130000001400000015000000160000001700000018000000190000001a0000001b0000001c0000001d0000001e0000001f000000200000002100000022000000230000002400000025000000260000002700000028000000290000002a0000002b0000002c00000b0000000000000000000000000000000000000000001f01000000000000000000000000ffffffff000039435555db42000000000000604100000000ffffffff000047435555db42000000000000000000006041ffffffff000039435555f742000000000000604100006041ffffffff000047435555f742000000008849a34802000000b415c908000000000000000000000000ffffffff000000008453a34802000000d415c908000000000000000000000000ffffffff00010000ec53a34802000000f415c908000000000000000000000000ffffffff000100005454a348020000001416c908000000000000000000000000ffffffff00010000bc54a3480200000000000000000000000000000000000000