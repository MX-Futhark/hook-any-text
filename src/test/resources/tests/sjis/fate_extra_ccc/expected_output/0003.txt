["———と。",
"呼ばれた気がして足を速めると、
そこには生徒会長でもあり友人でもある、
柳洞一成が立っていた。"]